apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue

```diff
@@ -144,6 +144,18 @@ const state = reactive({
                     ]).rawQueryStrings,
                 },
             };
+
+            const matchedJobIndex = collectorPageState.collectorJobStatus.findIndex((status) => status.collector_id === d.collector_id);
+            const recentJobAnalyze = collectorPageState.collectorJobStatus[matchedJobIndex]?.job_status.slice(-5) || [];
```

Comment:
suggestion: You can shorten this code using 'find' method. :)

```suggestion
            const matchedJob = collectorPageState.collectorJobStatus.find((status) => status.collector_id === d.collector_id);
            const recentJobAnalyze = matchedJob?.job_status.slice(-5) || [];
```